common/src/main/java/com/wynntils/features/user/InfoMessageFilterFeature.java

```diff
@@ -246,6 +258,36 @@ public void onInfoMessage(ChatMessageReceivedEvent e) {
                     return;
                 }
             }
+
+            if (toolDurability != FilterType.KEEP) {
+                Matcher matcher = NO_TOOL_DURABILITY_PATTERN.matcher(uncoloredMsg);
+                if (matcher.find()) {
+                    e.setCanceled(true);
+                    if (toolDurability == FilterType.HIDE) {
+                        return;
+                    }
+
+                    NotificationManager.queueMessage(
+                            new TextComponent("Your tool has 0 durability!").withStyle(ChatFormatting.DARK_RED));
```

Comment:
I think we've done it like this for other redirects, but these messages should really be i18n'd.

Reply:
https://github.com/Wynntils/Artemis/issues/626